dlio_benchmark/utils/utility.py

```diff
@@ -22,58 +22,66 @@
 from functools import wraps
 import threading
 import json
-from typing import Dict
-import pathlib
-from enum import Enum
+import tracemalloc
+from time import perf_counter
```

Comment:
why perf_counter?

Reply:
same as above, this is defined previously on line 247. i just moved that up.

happy to remove if this is not used at all